app/lib/surface/Lank.coffee

```diff
@@ -704,13 +704,14 @@ module.exports = Lank = class Lank extends CocoClass
 
   updateLabels: ->
     return unless @thang
-    blurb = if @thang.health <= 0 then null else @thang.sayMessage  # Dead men tell no tales
+    blurb = if @thang.health? and @thang.health <= 0 then null else @thang.sayMessage  # Dead men tell no tales, however non-alive can
     blurb = null if blurb in ['For Thoktar!', 'Bones!', 'Behead!', 'Destroy!', 'Die, humans!']  # Let's just hear, not see, these ones.
     if /Hero Placeholder/.test(@thang.id)
       labelStyle = Label.STYLE_DIALOGUE
     else
       labelStyle = @thang.labelStyle ? Label.STYLE_SAY
-    @addLabel 'say', labelStyle if blurb
+    if blurb
+      @addLabel 'say', labelStyle, @thang.sayLabelOptions
```

Comment:
Is this sayLabelOptions defined by the database game code?
Should we add some extra defensive checks here to ensure valid input?

Reply:
Good point, however for us it's enough this is an object here.